redbot/cogs/mutes/mutes.py

```diff
@@ -74,6 +74,8 @@ def __init__(self, bot: Red):
             "notification_channel": None,
             "muted_users": {},
             "default_time": 0,
+            "dm": False,
```

Comment:
Would it be beneficial to change this to `dm_on_mute` ? To be inline with other settings such as [dm_on_kickban](https://github.com/Cog-Creators/Red-DiscordBot/blob/7bb6e60c52134ed87b43841fabf0167a604cfe36/redbot/cogs/mod/mod.py#L57)

Reply:
Iirc this is in line with the warning cog's config key, however I am open for either.